Formula/infrakit.rb

```diff
@@ -0,0 +1,27 @@
+class Infrakit < Formula
+  desc "Toolkit for creating and managing declarative infrastructure"
+  homepage "https://github.com/docker/infrakit"
+  url "https://github.com/docker/infrakit.git",
+      :tag => "v0.5",
+      :revision => "3d2670e484176ce474d4b3d171994ceea7054c02"
+
+  depends_on "go" => :build
+  depends_on "libvirt" => :build
+  depends_on "pkg-config" => :build
```

Comment:
Are we sure about this one as well?

Reply:
Yea. Not sure where I get the idea from that this was needed.